### PR TITLE
Fix #14596: Remove pocket-mode

### LINKED ISSUE #14596: Epic: Remove Pocket mode
## Description

Now that we're no longer serving content for getpocket.com, we can remove the Pocket mode implementation and related pages. This will likely be split into a number of tasks, which can be defined below later when we're ready.

---

#### Issues
- [ ] #15124
- [ ] Front-end pocket mode removal

### PR DESCRIPTION
**Pull request by robhudson**
## One-line summary

Now that we're no longer serving content for getpocket.com, we can remove the Pocket mode implementation and related pages. 

- [ ] I used an AI to write some of this code.

## Issue / Bugzilla link

#14596 

## Testing

yes

### CODE REVIEW

**Comment by robhudson** [comment]
@alexgibson I attempted to keep the backend separate from removing the frontend. But removing the views required removing the urls, and removing the urls meant the templates couldn't resolve the referenced urls so I removed the templates. At that point it was just a matter of removing the `l10n-pocket, `css/pocket`, `img/pocket`, and `js/pocket` directories. So it ended up being easier to do it all in one.

File: bedrock/utils/braze.py [hunk 1]
```diff
@@ -1,69 +0,0 @@
...
-                event["properties"][f"subscriber_{name}"] = value
-
-        data = {"attributes": [attributes], "events": [event]}
-        headers = {"Authorization": f"Bearer {self.api_key}"}
-        resp = requests.post(self.track_user_api_url, json=data, headers=headers)
-        resp.raise_for_status()
-        return resp.json()
-
-
-client = BrazeClient(settings.BRAZE_API_URL_BASE, settings.BRAZE_API_KEY)
```
Comment (stevejalim) [inline bedrock/utils/braze.py:69]: Why was this file deleted?

File: bin/update_config.sh [hunk 2]
```diff
@@ -14,7 +14,7 @@ cd config_checkout
```
Comment (stevejalim) [inline bin/update_config.sh:14]: Change make sense, but wondering if it's a redundant file.

Is this file a remnant of the Deis days, I wonder...? @pmac will surely know

Comment (robhudson) [reply bin/update_config.sh:14]: A quick grep and I don't see it referenced anywhere, in bedrock or webservices-infra.

File: bedrock/utils/braze.py (hunk 1 above)
Comment (robhudson) [reply bedrock/utils/braze.py:69]: It was only used in the pocket pages for newsletter sign-ups. Mozorg uses basket (which then ends up in Braze).

**Review by stevejalim — APPROVED** [review]
I scanned the python related stuff an LGTM! r+!

File: bin/update_config.sh (hunk 2 above)
Comment (pmac) [reply bin/update_config.sh:14]: I don't think it was Deis, but it was for our k8s cluster before SRE took over hosting on their cluster. I'm confident we can delete this file.

**Review by alexgibson — APPROVED** [review]
This is great cleanup! r+wc

There's a doc here that can be removed too https://github.com/mozilla/bedrock/blob/main/docs/attribution/0006-pocket-analytics.rst

I'd also suggest pushing this branch to `run-integration-tests` before merging.

Nice work!

File: bin/update_config.sh (hunk 2 above)
Comment (robhudson) [reply bin/update_config.sh:14]: Thanks y'all!

**Review by janbrasna — COMMENTED** [review]
Only few irrelevant questions (without actually trying to look up the answers first):

File: bedrock/settings/base.py [hunk 3]
```diff
@@ -1167,7 +1145,6 @@ def _is_bedrock_custom_app(app_name):
 
 GA_ACCOUNT_CODE = ""  # DELETE ME: Deprecated?
 GTM_CONTAINER_ID = config("GTM_CONTAINER_ID", default="")  # NB: Will be used in both modes (bedrock and pocket).
-# Pocket mode will be running both GA UA and GA4 for a while going forward
 GOOGLE_ANALYTICS_ID = config("GOOGLE_ANALYTICS_ID", default="")  # NB: Not used in all Bedrock modes (Pocket only).
```
[the remark this answers is not in the document]
Comment (robhudson) [reply bedrock/settings/base.py:1153]: I can do a code search if those are still used.

[the remark this answers is not in the document]
Comment (janbrasna) [reply bedrock/settings/base.py:1153]: And/or set in the secrets.

[the remark this answers is not in the document]
Comment (robhudson) [reply bedrock/settings/base.py:1153]: Both `GA_ACCOUNT_CODE` and `GOOGLE_ANALYTICS_ID` appear to not be referenced anywhere in code or our env vars. I'll remove those.

File: docs/l10n.rst [hunk 4]
```diff
@@ -910,144 +910,3 @@ from the lang files then you can use the `add_active_locales` name in all of the
...
-.. important::
-    **If you are not using Pontoon/community translations, you do NOT need to create
-    this repo.**
-    Why? If the translations are done by the community (via Pontoon), there is a
-    possibility that not enough of the strings will be translated in order to render
-    the content in the relevant locale. We run a :abbr:`CI (Continuous Integration)`
-    task to determine whether a locale has enough translated strings to be considered
-    'active'. At the moment, only Mozorg uses this pattern. The Pocket-mode translations
-    do not have their activation measured because their translations come entirely from
-    a vendor and we expect the Pocket strings to be 100% translated.
```
[the remark this answers is not in the document]
Comment (robhudson) [reply docs/l10n.rst:993]: I added this back and re-wrote it more as informational docs about how the l10n pipeline works instead of instructions on adding a new pipeline. Thanks for pointing out the usefulness of this section.

**Comment by robhudson** [comment]
Integration tests passed: https://github.com/mozilla/bedrock/actions/runs/11296732003
After bedrock-deploy was updated here: https://github.com/mozilla-sre-deploy/deploy-bedrock/pull/25
